packages/shared/lib/services/connection.service.ts

```diff
@@ -579,7 +581,17 @@ class ConnectionService {
     public async listConnections(
         environment_id: number,
         connectionId?: string
-    ): Promise<{ id: number; connection_id: string; provider: string; created: string; metadata: Metadata; active_logs: ActiveLogIds }[]> {
+    ): Promise<
+        {
+            id: number;
+            connection_id: string;
+            provider: string;
+            created: string;
+            metadata: Metadata;
+            active_logs: ActiveLogIds;
+            connection_config: ConnectionConfig;
+        }[]
+    > {
```

Comment:
Should probably be `ConnectionList[]` no? (the type name is confusing but can be changed later)

edit: nevermind the type is only in the frontend 😞